api/clients/v2/disperser_client.go

```diff
@@ -139,16 +138,11 @@ func (c *disperserClient) DisperseBlob(
 		return nil, [32]byte{}, api.NewErrorInternal("uninitialized signer for authenticated dispersal")
 	}
 
-	// TODO(hopeyen): uncomment this after the accountant is implemented
-	// if c.accountant == nil {
```

Comment:
hmm shall we keep this check? 
what happens if `initOncePopulateAccountant` fails and client tries to call this method again?

Reply:
hmm I guess if the client tries this method again after the init failed, all subsequent calls will return the same error from the first attempt. I think it would be the same behavior as adding this check back?